composer.json

```diff
@@ -9,16 +10,19 @@
     ],
     "minimum-stability": "stable",
     "require": {
-    	"php": ">=5.3.0",
-        "guzzlehttp/guzzle": "3.8.1"
+    	"php": ">=5.5.0",
```

Comment:
Can we allow 5.4 users to keep using it now that we have the adapter pattern in place? Is there any harm? I think I'm fine with either decision, just curious.